src/common/components/Recaptcha/Recaptcha.tsx

```diff
@@ -94,7 +95,9 @@ export const Recaptcha = ({
 
         if (data?.success === true && isValidAction(data?.action)) {
           if (data.score < 0.5) {
-            $log.warn(`Captcha score was below the threshold: ${data.score}`)
+            const errorMessage = `Captcha score was below the threshold: ${data.score}`
+            $log.warn(errorMessage)
+            datadogRum.addError(new Error(`Error submitting purchase: ${errorMessage}`))
```

Comment:
To help with debugging, you could add context to the error logs. But this isn't required.
```suggestion
            datadogRum.addError(new Error(`Error submitting purchase: ${errorMessage}`),
            { context: 'Recaptcha flow', errorCode: "lowScore" }
            )
```